### PR TITLE
added fix of mistakes

### PR DESCRIPTION
**Pull request by taras-oleksiuk**
fixed checkstyle mistakes

### CODE REVIEW

File: src/main/java/core/basesyntax/HelloWorld.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    private void sayHello(String hello) {
+        System.out.println("Hello, Mate! I am happy to see you! This class looks weird! You"
+                           + "r task is to fix the  Your task is to fix the checkstyle in this fil"
+                + "e. Your goal is to run `mvn clean package` and receive `BUILD SUCCESS` message");
+        System.out.println(hello);
+    }
+
+    private void initializeVariables() {
+        name = "Hello mates!";
```
Comment (Resci) [inline src/main/java/core/basesyntax/HelloWorld.java:37]: `Hello mates!` is a name?

File: src/main/java/core/basesyntax/model/Cat.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+public class Cat {
+    private int age;
+    private String name;
+    private String ownerName;
+
+    @Override
+    public String toString() {
+        return "Cat{ age=" + age + ", name='" + name + '\''
+                + ", ownerName='" + ownerName + '\'' + '}';
```
Comment (Resci) [inline src/main/java/core/basesyntax/model/Cat.java:11]:
```suggestion
        return "Cat{" 
        		+ " age=" + age 
        		+ ", name='" + name + '\''
                + ", ownerName='" + ownerName + '\'' + '}';
```

**Review by Resci — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/HelloWorld.java (hunk 1 above)
Comment (taras-oleksiuk) [reply src/main/java/core/basesyntax/HelloWorld.java]: The variable has been renamed as welcome.

File: src/main/java/core/basesyntax/HelloWorld.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String welcome = "";
```
Comment (Resci) [inline src/main/java/core/basesyntax/HelloWorld.java:8]:
```suggestion
    private String greeting;
```

File: src/main/java/core/basesyntax/HelloWorld.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+                result += i + " ";
+            }
+        }
+        return result;
+    }
+
+    private void sayHello(String hello) {
+        System.out.println("Hello, Mate! I am happy to see you! This class looks weird! You"
+                           + "r task is to fix the  Your task is to fix the checkstyle in this fil"
+                + "e. Your goal is to run `mvn clean package` and receive `BUILD SUCCESS` message");
```
Comment (Resci) [inline src/main/java/core/basesyntax/HelloWorld.java:32]:
```suggestion
        System.out.println("Hello, Mate! I am happy to see you! This class looks weird! You"
                           + "r task is to fix the  Your task is to fix the checkstyle in this fil"
                           + "e. Your goal is to run `mvn clean package` and receive `BUILD SUCCESS` message");
```

**Review by Resci — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/HelloWorld.java (hunk 3 above)
Comment (taras-oleksiuk) [reply src/main/java/core/basesyntax/HelloWorld.java]: Created a new variable name "greeting" without assignment ""

**Review by Resci — CHANGES_REQUESTED** [review]
Fix checkstyle
image

**Pull request closed without merging**